code/modules/mob/living/carbon/human/human.dm

```diff
@@ -924,10 +924,10 @@
 
 	return Clamp(., -1, 2)
 
-
 /mob/living/carbon/human/IsAdvancedToolUser()
-	return 1//Humans can use guns and such
-
+	if(M_HULK in mutations) //We hulked out

```

Comment:
This already existed for guns, I simply ported it over to human's AdvancedToolUser check